CONTRIBUTING.md

```diff
@@ -72,28 +41,57 @@ for ways to improve the API to make it easy to work with the GitHub API.
 
 ## Making Changes
 
-When you're ready to make a change, 
-[create a branch](https://help.github.com/articles/fork-a-repo#create-branches) 
-off the `master` branch. We use `master` as the default branch for the 
+When you're ready to make a change,
+[create a branch](https://help.github.com/articles/fork-a-repo#create-branches)
```

Comment:
This anchor doesnt actually exist on that page...  Did you actually mean it to be https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/#creating-a-branch 

Or perhaps the "Next Steps" anchor on the "fork" page?
https://help.github.com/articles/fork-a-repo/#next-steps